content/work/index.rst

```diff
@@ -26,13 +23,13 @@ process:
   - title: 'Accessible <i>&</i> Responsive'
     body: |
       We use the accessible open web stack:
-      semantic HTML5, unobtrusive JavaScript, and advanced CSS —
+      semantic HTML5, unobtrusive JavaScript, and advanced CSS –
       backed by Python & Django on the server.
 services:
   - title: 'We have partnered with'
     data:
       - 'Educators'
```

Comment:
Is there a reason these are in single quotes and the "We provide" list below this does not have quotes?
![quotes](https://cloud.githubusercontent.com/assets/1581694/24625294/2b6143e2-1874-11e7-891e-8b4f0ff2bb73.jpg)